DOCS-3747: Add example for querying from locations (#4117)
viamrobotics/docs: docs/data-ai/data/query.md

{{% tablestep number=2 %}}
**Run your query**

> This example query returns 5 readings from all machines in your current organization where the component is called `my-sensor`:

Reviewer: Still sounds confusing--maybe this? Not sure if it's the most recent 5 or not, but either way I think saying from any components called this makes more sense.
```suggestion
This example query returns the last 5 readings from any components named `my-sensor` in your organization:
```

    "readings": {
      "pm_2.5": 9.3,
      "units": "μg/m³",
      "pm_10": 11.5
    }
  },
  "tags": [
    "air-quality"
  ]
}
]
```

{{% /tab %}}
{{% tab name="MQL" %}}

```sh {class="command-line" data-prompt="$" data-output="16-80"}
[
  {
    $match: {
      component_name: "PM_sensor"
    }
  },{
    $limit: 2
  }, {
    $project: {
        time_received: 1,
        data: 1,
        tags: 1
    }
  }
]
[
{
  "time_received": "2024-07-30 00:04:02.144 +0000 UTC",
  "data": {
    "readings": {
      "units": "μg/m³",
      "pm_10": 7.6,
      "pm_2.5": 5.7
    }
  },
  "tags": [
    "air-quality"
  ]
},
{
  "time_received": "2024-07-30 00:37:22.192 +0000 UTC",
  "data": {
    "readings": {
      "pm_2.5": 9.3,
      "units": "μg/m³",
      "pm_10": 11.5
    }
  },
  "tags": [
    "air-quality"
  ]
}
]
```

{{% /tab %}}
{{< /tabs >}}

{{% /expand %}}
{{% expand "Click to see an example that returns a count of records that match a component name from a specific location." %}}

{{< tabs >}}
{{% tab name="SQL" %}}

```sh {class="command-line" data-prompt="$" data-output="3-80"}
SELECT count(*) FROM readings
WHERE component_name = 'PM_sensor' AND location_id = 'ab1cd23e4f'
[

> {
>   "_1": 111550
> }

Reviewer: Maybe?
```suggestion
  {
    "_1": 111550
  }
```
